tools/pipeline-runner/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN apk add --no-cache \
   openjdk11-jre-headless \
   openssl \
   util-linux \
+  perl-xml-xpath \
```

Comment:
we already have `libxml2-utils` so shouldnt need this - i also cant see where it is used